lib/src/license_detection/crc32.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+part of 'license_detector.dart';
```

Comment:
I think this could be an independent library, doesn't need to be part of the license_detector.dart

Reply:
Yes but we did not want to expose the crc32 function and similar other functions outside of license_detection and hence made it part of the `license_detector`.

Reply:
Making this a library doesn't expose it as public API. (that happens only with exports)